website/docs/basics/basics.md

```diff
@@ -3,3 +3,27 @@ id: basics
 title: Basics
 sidebar_label: Basics
 ---
+
+Before WordPress landed in 5.0 release it way mostly PHP based project with some JavaScript and CSS in the themes. Not it it a bit different, we now have PHP in the core, JavaScript (specifically Reach) in the Block Editor and some styles in the editor part and the theme part.
```

Comment:
```suggestion
Before WordPress landed in 5.0 release, the themes were mostly PHP based project with some JavaScript and CSS. Now it's a bit different, because  now we have PHP in the core, JavaScript (specifically React.js) in the Block Editor and some styles in the editor and in the theme parts of the website.
```